drivers/sensor/hp206c/hp206c.h

```diff
@@ -73,7 +73,7 @@
 /* end of default settings */
 
 struct hp206c_device_data {
-	const struct device *i2c;
+	const struct device *i2c_bus_dev;
```

Comment:
Why did you rename this? The change adds some unnecessary noise to the diff

Reply:
removed the variable name change.